coralproject/talk
# Updating embed link style to match designs.

## What does this PR do?

Updates the style of the embed link to match designs.

## How do I test this PR?

1. `npm start`
2. Go to `/admin`
3. Click Configure -> Embed Link
4. Link should be styled to match: https://www.pivotaltracker.com/file_attachments/71477487/download?inline=true
5. Copy button should work and should display confirmation.

@coralproject/tech
